### DISCUSSION TITLE
Support for Control Templates

### DISCUSSION THREAD

**Discussion by vhugogarcia** [dotnet/maui#2316] [Q&A] [answered]
Hello friends,

Anybody knows if .NET MAUI is going to support Control templates just like Xamarin.Forms? https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/templates/control-template

Additionally, Do you know if we can show a control template based on a platform? For example: If Phone then show X control template, if tablet then show Y control template, if TV then show Z control template.

Thanks in advance.

**Comment by vhugogarcia** [comment, marked as answer]
This has been implemented.Thank you all 🙂

**Reply by janseris** [reply]
Can you post link about the solution?
